ci/docker/Makefile

```diff
@@ -33,7 +33,7 @@ all: fedora centos
 fedora: fedora31 fedora32 fedora33 fedora34
 
 fedora31:
-	docker build -t ats_$@ --build-arg OS_VERSION=31 --build-arg OS_TYPE=fedora yum/
+	docker build -t ats_$@ --build- OS_VERSION=31 --build-arg OS_TYPE=fedora yum/
```

Comment:
Someone might see this is broken and accidentally fix it. Please add comment explaining why it should be broken.